_projects/sparcv8leon3-gr712rc-board/board_config.h

```diff
@@ -59,6 +59,13 @@
 #define UART4_ACTIVE 0
 #define UART5_ACTIVE 0
 
+#define UART0_DMA 0
```

Comment:
To run DMA on sparcv8leon3-gr716-mimas, we must enable it on UART0. After that everything works perfectly.

Reply:
You've marked the wrong file :) (for `gr712rc`). DMA is enabled on the console UART1 on `gr716-mimas` (https://github.com/phoenix-rtos/phoenix-rtos-project/blob/34b7b7dd130a297d2fcc7222d2d027163159dca7/_projects/sparcv8leon3-gr716-mimas/board_config.h#L63), should it be on UART0 instead?

Reply:
You are right :+1: Yes, it should be on UART0. My bad.

Reply:
Done :+1: